### DISCUSSION TITLE
Tues/Thurs. Coffee Chat backpocket & intro topics

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#492] [New Ideas]
If you've been to Virtual Coffee, you know that every Tuesday and Thursday when we meet up, we prioritize what our members want to talk about, but always have a back pocket topic. 

If you have a backpocket topic, you'd like to add to the list, we'd love to hear it! Drop it in the discussion below.  We try to use topics that are open to a wide community of people and allow for discussion.

**Comment by BekahHW** [comment]
In the past, we've covered these topics:

# Topics

_Virtual Coffee started out of a need for community, out of a need for conversation. We're not a meetup. We believe in a person-first approach, where we recognize that each person is a complex human being who isn't defined by their tech stack, etc. Conversations are often focused on tech, because that's what most of us do, but they don't have to be limited to tech._

## Here are some topics we frequently visit at VC:

- education
- community
- communication (and misunderstandings)
- interviewing
- mental health
- worklife/balance
- hobbies
- navigating challenges
- wellness
- what excites you
- growth mindset
- networking
- job environments
- asking for help
- documentation
- accessibility
- design
- open source
- physical wellness
- ergonomics
- social equity
- core competency and stretch competency

Please feel free to use these as back-pocket topics at any time. And talk about what feels comfortable to you.

## Topics with Addition prompts
### Challenges
- what’s a challenge you’ve faced and overcome?
- How do you approach them?
- What happens when you get stuck?
- What have you learned from it and been able to apply in other situations?

### Documentation.
- Why is documentation important? What makes good documentation? Do you have any examples of good documentation? How do you think companies/products can improve documentation?


### Education
- How did you learn to code
- we’re always learning, what are your favorite places to learn?
- what gaps have you seen when trying to learn something new?
- does your job provide support for continuing education

### Community
- What does this mean to you?
- what are some of your favorite communities and why?
- what challenges have you faced as part of a community?
- Do you think community is important?

### Communication (and misunderstandings)
- How do you best communicate?
- How do you handle challenging conversations?
- Have you ever tried to break some bad communication habits? How did that go?
- How does your personality influence your communication style?
- How do you deal with misunderstandings?
- Remote v. in person communication
- Async v. sync communication

### Interviewing
- The good, the bad stories
- How to make sure you interview the company interviewing you
- How to be a good interviewer
- How to “study” for the interview
- Tips for interviewing
- Do you need to unwind or decompress after an interview? What are your favorite go-tos to recover?


### Motivation 
- What motivates you?
- How do you deal with it when you can’t find motivation?
- What are some ways you’ve been motivated by others?
- What are things that de-motivate you?
- How could we help motivate you today?

### Being Ready
- How do you know when you're "done" learning and ready to code? 
- Does this change over time? 
- What things give you confidence that you are "ready"?
- What's the difference between learning about a new framework, for example, and using it?
- What advice do you have for people who feel "stuck"?

### Conferences
- What makes a good conference or meetup?
- What are some strategies to make them successful?
- What is your main goal when you go to a conf/meetup?
- Long-term impacts of attending?
- What's the difference between in-person and virtual events?
- What have made/could make the best ones?

### Project-Induced Anxiety
- How do you deal with anxiety caused by a project?
- How do you communicate your anxiety to your supervisors
- What do you do when dealing with the project interferes with team progress?
- How do you problem solve when the whole team recognizes that the codebase is a challenging mess?

### Healthy Habits, Happy Devs
- How do you recharge and recover? This month's challenge is all about nourishing our bodies, minds, and spirits so that we can become healthier developers. Let's talk about the things we do to become healthier devs.
- What are some habits that make you a healthier dev? This can be mind and body centered (drink, move, read, meditate, rearrange your work station) or code centered (review your ReadMe, clean your code, refresh your GitHub repo) or both.

### Learning
- How do you decide what topics to learn deeply or projects to focus on? 
- Do you have a process? 
- How do you avoid doing a lot of things shallowly?

### Interviewing people
- how do you prepare, 
- do you have a list of go-to questions, 
- how do you prepare the test you'll give (whatever the kind of test)

### Mental Models
*inspired by Just JavaScript*, ch. 1
> These approximations of how something works in your head are known as “mental models.”
- How do you develop mental models?
- Do you notice them as you're coding?
- Do you ever have to rethink and recalibrate your mental models? If so, how do you reinforce the changes?


### Trends
- What trends in technology are you skeptical or uncertain of?
- How do you stay up to date on technology trends? 
- What are your favorite sources?


### Career Motivation
What motivated you to pursue a career in technology?  Is it the same thing that keeps you motivated now?

### Transferable Skills
- what are some transferrable skills you bring to tech, either from a previous career or from other parts of your life?”
- What have been your best experiences with working and learning? What made those experiences good? How can you carry what you've learned in those experiences to new experiences?

### Growth
- How do you know when you’ve outgrown a role?” “How do you create opportunities for growth for yourself?” “How do you know when a job will really offer you opportunities to grow?”
- What's your favorite non tech topic to study or learn? Has it helped you in your tech career?

### Industry
What are the most painful barriers to entry for being "in tech"? What should we do to reduce these blockers?
Does having close family or friends in tech help or hinder you in your tech journey?


### User experiences. 
- As developers, we create technologies, but in our daily lives, we're consumers like everyone else. What user experiences online frustrate/baffle you, and how do those inform your decisions as a developer? Conversely, have you ever experience a UI/UX that brought joy or relief?

### OSS
- What are the most painful barriers to entry for open source contributions? What should we do to reduce these blockers? What have been your favorite OSS repos to contribute to?

**Comment by vintprox** [comment]
_Inspired by faults of [Flask documentation](https://flask.palletsprojects.com/en/2.0.x/) on background of Django's job market boom._

Have you read any epic documentation that changed your perception of technology? How much do you think documentation of tool affects the public perception of this tool overall? Does it mostly affect job market requiring this tool or something else?

- [x] Discussed? (2024.10.24)

**Comment by rek990** [comment]
- [x] Regarding the Learning topic: Are you a note-taker? Do you take notes on a computer, tablet, pen/paper? Which organizational tools (e.g., Notion, Org-mode, etc.) work for you, and which ones do not?

**Comment by Cerchie** [comment]
I'd love to add test-driven development! Not sure what umbrella topic to put it under

**Comment by BekahHW** [comment]
### Burnout
- We hear a lot about burnout in tech. Do you think this is industry-specific? What contributes to burnout? How do we navigate burnout? When is changing jobs a solution to burnout? How can we reduce burnout in the industry?

### Ethics in Tech
- With tech moving so quickly, it's nearly impossible for ethics to keep up. What does tech ethics mean to you? What do you think the impact could be? Do you think we should build tech ethics into bootcamps, self-taught courses, new employee onboarding?

### Gatekeeping
- Unfortunately, gatekeeping is a problem we see in tech. Gatekeeping is when a person/organization--intentionally or unintentionally--discourages or prevents someone from achieving their goals. What does unintentional gatekeeping? What does intentional gatekeeping look like? Is gatekeeping always a bad thing?

**Comment by meg-gutshall** [comment]
## Intros

- [x] What's the song you listen to when you need to calm down?
- [ ] What's the corniest joke you ever heard?
- [ ] Who is your tech role model?
- [ ] Do you have a personal hero or someone you look up to?
- [x] What's the strangest gift you've ever been given or gifted to someone else? (Things to consider: White elephant parties)
- [x] What's the weirdest thing you've ever bought?
- [ ] What has been your best birthday memory to date?
- [x] Who would you want to narrate your internal dialog? Whose voice would you **_hate_** to have inside your head? (Things to consider: Morgan Freeman, Gilbert Gottfried, cartoon characters)
- [x] Would you change your height if you could?
- [ ] Would you eat a tennis ball for $1,000?
- [x] You have the chance to get a free, non-fading scratch-and-sniff tattoo. What would you get inked, if anything?
- [ ] If you were a supervillain, what would be your name and/or superpower? (Things to consider: **The Brown Note** is already taken)
- [ ] If you had a catch phrase, what would it be?/What's a phrase you're known for saying?
- [ ] What fictional world would you like to live in? Assume you're somehow connected with the main characters of the show/movie/book/etc. this world originates from.
- [ ] Who would you like to see on a celebrity roast?
- [ ] Which famous icon (living or deceased) would you be amused to receive a drunk phone call from?

### Would you rather...

- [ ] Make the sounds of a caveman when straining physically **- _OR_ -** Make the sound of the _Jeopardy_ theme when straining mentally? (Things to consider: debugging, working out)
- [ ] Have to communicate solely in baby talk **- _OR_ -** Always speak like you're reading a telegram? (Things to consider: when you "gotta go boom-boom," when you're ending a sentence **STOP**)
- [ ] Have invisible skin **- _OR_ -** Have everyone else's skin appear invisible to you? (Things to consider: modeling for biology textbooks, diagnosing illnesses, becoming the world's best surgeon)
- [ ] Always have to wear NFL referee garb **- _OR_ -** Always have to wear a wizard's robe and hat? (Things to consider: stripes, irate sports fans, Harry Potter conventions)
- [ ] Speak in the voice of a possessed Linda Blair in _The Exorcist_ whenever talking to cashiers or retail workers **- _OR_ -** Not be able to speak to them at all? (Things to consider: "The power of cash compels you!", miming skills)
- [ ] Be able to jump like Dr. J in the ABA, but always have to wear those short shorts he had **- _OR_ -** Have the eloquence of Thomas Jefferson, but have to wear colonial garb and wig?
- [ ] Snore the sound of a dial-up modem **- _OR_ -** Be allergic to smooth jazz?
- [ ] Be able to fast-forward life **- _OR_ -** rewind it?
- [ ] Be able to insist on paying for the check, but never actually get stuck with it **- _OR_ -** Know exactly what the person on the other end of the phone looks like simply by hearing their voice?
- [ ] Have the power to switch you emotions on and off **- _OR_ -** Be able to fully comprehend written material just by sniffing the words?
- [ ] Be a brilliant essayist, but have to wear a matching set of wristbands and headband at all times **- _OR_ -** Have heightened Stratego intuition, but speak in an accent that's clearly not yours when asked to repeat yourself?
- [ ] Fight 20 miniature raccoons **- _OR_ -** Fight 1 giant goose? (Things to consider: The raccoons are 1/4 their normal size and the goose is 4x its normal size)
- [ ] Live in a world where Teletubbies are a common species of creature that lives in the wild **- _OR_ -** Live in a world where there are evil "Bizarro" arch-enemy versions of ourselves? (Things to consider: hunting)
- [ ] Live in a world where the pledge of allegiance is changed to the lyrics to "Eye of the Tiger" **- _OR_ -** Live in a world where there is a rapper-like East Coast/West Coat feud of mimes?
- [ ] Live in a world without skin moisturizer **- _OR_ -** Live in a world without cream cheese?
- [ ] Live in a world without Sinbad **- _OR_ -** Live in a world without Men without Hats' "Safety Dance"?
- [x] Have a lake named after you **- _OR_ -** Have a popular children's multivitamin shaped in your image?
- [ ] See Lincoln and Washington debate **- _OR_ -** See them play a game of one-on-one basketball?
- [ ] You can add one rule to make soccer more exciting. Would you rather... Have grazing livestock scattered throughout the field **- _OR_ -** Change "headers" to "butters"? (Things to consider: Possible PETA protests, less concussions)
- [ ] You can add one rule to make basketball more exciting. Would you rather... Institute a 4-foot maximum height rule **- _OR_ -** Bring back the short shorts uniforms?
- [ ] You can add one rule to make baseball more exciting. Would you rather... Light the bases on fire (excluding home plate) **- _OR_ -** Allow peg-outs? (Things to consider: Much, **much** more padding)
- [ ] You can add one rule to make boxing more exciting. Would you rather... Have a round where the boxers dance-off **- _OR_ -** Require that whoopee cushion-like devices be placed in the boxers' gloves?

**Comment by BekahHW** [comment]
From slack

## Intros

- [ ] What was the most recent costume you wore, and for what event?
- [ ] If you could demand any music artist release a new album next month, who would it be?
- [x] What is your favorite plant? Decorative, food, whatever?
- [x] What was your favorite childhood toy?
- [ ] If you could get paid to take a week off work to design a new Lego set, what would you build?
- [x] What was your favorite school project or report you had to do?
- [x] What is your favorite creativity outlet (making music, doodling, knitting, etc)?


## Backpocket

- [x] If you had a month to build a website that did anything you want and was free for everyone to use what, what would your website do?
- [x] Let's talk design patterns -- what are they? How do you use them?
- [x] If you suddenly had to switch into a different field within tech, what would you try (DevOps, QA, frontend, backend, data, etc)?
- [x] What subject/book/topic besides tech did you learn that is helping you with tech work today?

**Comment by SuperRoach** [comment]
## Intros

- [ ] Travelling or driving is something all of us do, and at some points have to put up with. What do is your pet peeve with travelling, and how would you deal with it?
- [ ] What is the best easter egg or surprise you have discovered?
- [ ] If you could travel back in time to yourself at ten, what advice would you give yourself?
- [ ] What would be the worst password you could come up with?
- [ ] What is the craziest input device you've used?
- [ ] What is the strangest prize you have been awarded?

**Comment by ethanfreire** [comment]
## Intros

- [ ] Which discontinued tech gadget do you wish came back?
- [ ] What is your favorite "third place" to go to?
- [ ] Close your eyes and imagine an apple. What do you see?
- [ ] Do you sleep with a night light on?
- [x] Which genre of horror movie scares you the most? The least?
- [ ] Which seasonal dish do you wish was available year-round?
- [ ] If you could have your favorite local restaurant chain available anywhere you travel, which would you pick?
- [ ] Are there any unconventional food combinations you recommend that others try?
- [x] Which animal do you wish you had the power to tame instantly?
- [ ] Which discontinued food product do you wish came back?
- [ ] If you had mind-reading power, who would you use it on?
- [ ] If you could add a new feature to your phone, what would it do? (Think: Like the watch in the movie _Spy Kids_  that can do everything except tell time)
- [x] Who would you like as a mentor or a coach?
- [ ] If you could delegate a task to someone, what would it be?
- [ ] What restaurant experience would you like to try out? (Link: [https://www.rd.com/list/weird-restaurants/](https://www.rd.com/list/weird-restaurants/))
- [ ] If you had a ticket for 100% probability of an event happening. What would you spend it on?
- [x] If you had a magical orb to see the multiverse, which universe would you like to peek into?
- [ ] If you could be in multiple places simultaneously, where else would you like to be?
- [ ] What music do you listen to help get you into "focus/work" mode?
- [ ] When playing multiplayer games, which [Bartle player type](https://en.wikipedia.org/wiki/Bartle_taxonomy_of_player_types) are you?
- [ ] Are there any English-language oddities/inconsistencies you find funny or interesting? 
- [ ] Do you have any tips on how to deal with long flights?
- [ ] What kind of sleeper are you? (Link: [https://www.peacockalley.com/blogs/the-linenry/types-of-sleepers-what-kind-of-sleeper-are-you](https://www.peacockalley.com/blogs/the-linenry/types-of-sleepers-what-kind-of-sleeper-are-you))
- [ ] If you had the power of telepathy to control nanobots, what would you make them do? (Think: Like in the movie _Big Hero 6_)
- [ ] If you had the power to control your passions, what would you like to be passionate about?
- [ ] If you had a perfect AI clone of your mind, what is the first task you would give it to do?
- [ ] If you could buy a perfect AI clone of someone's mind, whose would you buy? What is the first question you'd ask them? 
- [x] If you could have any voice actor's voice for a day, whose would you pick?
- [ ] What random thought lives in your head rent-free?
- [x] What show that was canceled early do you wish got another season and or movie? 
- [ ] If you could speak with an accent. Which accent would you choose? Link: https://www.enjoytravel.com/en/travel-news/places-to-visit/worlds-sexiest-accent
- [x] If you had a pet and they could have a job. What job would you like them to have? 

## Backpocket

- [ ] How would you change the technical interview process?
- [ ] What is a dream course you wish you could have taken? Who would you like to be your instructor?
- [ ] What are your tips on how to improve self-control? Link: https://www.wsj.com/articles/how-the-brain-uses-glucose-to-fuel-self-control-1417618996
- [ ] What's your favorite life hack? 
- [ ] Which self-help advice do you agree with the most? Disagree? (Link: [https://www.highexistence.com/7-popular-self-help-tips-fking-life/](https://www.highexistence.com/7-popular-self-help-tips-fking-life/))
- [ ] Any tips on providing adequate emotional support?
- [ ] What is your favorite tip to achieve a tidy home? (Link: [https://montanahappy.com/secrets-for-a-tidy-home/](https://montanahappy.com/secrets-for-a-tidy-home/))
- [ ] A common problem-solving strategy when coding is understanding the problem. What are some strategies you use to understand the problem? (Link: [https://www.henricodolfing.com/2018/05/understanding-your-problem-is-half.html](https://www.henricodolfing.com/2018/05/understanding-your-problem-is-half.html))
- [ ] When doing a code review, what is your first look for? 
- [x] Are there any software tools, Chrome extensions, or apps you recommend?
- [ ] Do you have any advice on common pitfalls for new developers to avoid? 
- [x] If you could change how your work schedule is structured, what changes would you make? (Link: [https://www.indeed.com/career-advice/career-development/alternative-work-schedules](https://www.indeed.com/career-advice/career-development/alternative-work-schedules))
- [ ] Do you have a growth or fixed mindset? (Link: [https://www.betterup.com/blog/fixed-mindset](https://www.betterup.com/blog/fixed-mindset))
- [ ] Companies love to say they promote a culture of diversity and inclusion. In your workplace, what changes has your company made that you find helpful in promoting a culture of diversity and inclusion?
- [ ] Every workplace is filled with both neurotypical and neurodivergent folks. Neurotypical employees: Any workplace advice for neurodivergent employees? Neurodivergent employees: Any workplace advice for neurotypical employees? (Link: [https://www.askamanager.org/2019/03/ask-the-readers-how-to-succeed-at-work-when-youre-not-neurotypical.html](https://www.askamanager.org/2019/03/ask-the-readers-how-to-succeed-at-work-when-youre-not-neurotypical.html))
- [ ] How do you manage your to-do list? Do you have any tips on how to organize a to-do list? Do you have a favorite to-do list app?
- [ ] Some people create personal rules to simplify decision-making and free up mental bandwidth. What are some unique rules that you've developed around task management/getting things done? 
- [x] We must take in boatloads of information daily, but where does it go? When we don't know what to do with it, we feel overwhelmed. How do you deal with information overload?
- [ ] Do you have any tips on getting consistent, deep sleep each night?
- [ ] Do you have any tips on how to tame stress?
- [x] Do you have any tips on understanding or debugging a large codebase, given it has documentation? Does documentation work for you?
- [ ] Suppose you could enter a world for a day based on one of your dreams. Which dream would you pick? 
- [ ] If you could interview someone, who would you interview? 
- [ ] Any tips to avoid overengineering your application?
- [ ] During the interview process, what question / how did you phrase your question to know of a potential hidden red flag when working for this company? 
- [ ] What is your favorite targeted interview question to ask to gain company insight from the hiring manager, software engineer, team manager, CTO,  human resources lead, or recruiter? 
- [ ] What are some of your workplace deal breakers? Link: https://www.recruiter.com/recruiting/bamboohr-study-reveals-top-5-workplace-deal-breakers-infographic/
- [ ] What is your faith or belief?
- [ ] Do you consider yourself spiritual or religious?
- [ ] What things do you believe in that give meaning
 to your life?
- [ ] What is your source of hope? 
- [ ] What is your go-to source for comfort?
- [ ] Are you part of a spiritual or religious community? Is this of support to you, and how?
- [ ] Is there a person or group of people you really love or who are really important to you?
- [ ] What fuels your energy drive to keep you going? 
- [x] What is your go-to source of joy? Memes, social groups, accounts to follow? 
- [x] When faced with failure, what makes you bounce back up and try again? 
- [ ] What aspects of your spirituality or spiritual practices do you find most helpful to you personally? (e.g., prayer, meditation,
 reading scripture, attending religious services, listening to music, hiking, communing with nature, journaling, traveling, voice memos )
- [ ] Where is your dream vacation destination? What is the first thing you want to do there? How long would you want to stay there? 
- [ ] Who was the first person to inspire you to begin your developer learning journey? 
- [ ] How do you currently love yourself? Practicing self-love: https://www.baileyopsal.com/awaken-powerful-self-love/
- [ ] What are some things you enjoy doing? Where in your life are you doing these things?
- [ ] What do I want? What are my choices? What am I missing or avoiding? [Change Your Questions, Change Your Life](https://a.co/d/igxy1Nd) 
- [ ] "What's the ONE Thing I want to do this year?" and then "What's the ONE Thing I want to do this month?" "What's the ONE Thing I want to do this week?" "What's the ONE Thing I want to do today?" Link: https://www.hackingyouradhd.com/podcast/breaking-down-the-one-thing
- [ ] Growing up, what messages about money did adults around you instill in you? How did you approach your own money as you entered adulthood? Link: https://moneymentors.ca/money-tips/how-to-shift-your-money-mindset/
- [ ] If you could permanently change the price of one item to $1, what would it be?
- [ ] In your opinion, What was hugely hyped up but flopped?
- [ ] In your opinion, What profession is held in high regard that shouldn't be?
- [ ] Bill Gates said, "I will always choose a lazy person to do a difficult job because a lazy person will find an easy way to do it." What's a real-life example of this?
- [ ] What free things online should everyone take advantage of?
- [ ] With all of the negative headlines dominating the news these days, it can be not easy to spot signs of progress. What makes you optimistic about the future?
- [ ] How do you make friends as an adult if you don't drink?
- [x] What are subtle red flags you look for at a job interview that says "working here would suck"?
- [ ] What are some relationship GREEN flags?
- [ ] What is clearly a scam but is so normalized that people don’t notice?
- [ ] What is the biggest lie that, as a society, we still believe to be true?
- [ ] Richard Feynman said, “Never confuse education with intelligence; you can have a Ph.D. and still be an idiot.” What are some real-life examples of this?
- [ ] What productivity lie do you believe is true? Link: https://www.inloox.com/company/blog/articles/myths-busted-the-6-biggest-productivity-lies-we-tell-ourselves/
- [x] How do you manage your work and personal life? Do you have any apps/tips/tricks/automation/life hacks you recommend? When life gets out of control, how do you regain control? 
- [ ] Who do you want to help?
- [ ] Who do you want to spend your time with today?
- [ ] Whom do I admire most in the world?
- [ ] What are your needs? What are your wants? What don't you need? What don't you want? 
- [ ] What gives you purpose? 
- [ ] What activities put you in a flow state?
- [ ] How do you define a “good life/balanced life?”
- [ ] Are you in the right location to do what you want to do in work or life?
- [ ] What are you willing to sacrifice to achieve your goal in 2024?
- [ ] What are your most significant accomplishments in life so far?
- [ ] If you had to pick one word/phrase to describe your life in 2023, what would you choose? What word/phrase would you like to pick for 2024? 
- [x] What activity would you work for free for ?
- [ ] What is a mistake you've made? What did you learn from it? Have you made the same mistake since? 
- [ ] Do you have courage in regards to ....[INSERT TOPIC HERE]?
- [ ] Do you interact with people that have different lived experiences than you?
- [ ] How do you ask for a one-on-one meeting with your manager? What is the structure of your one-on-one meeting? 
- [ ] So, Post-college education, [where do you go to interact](https://www.businessinsider.com/rich-poor-americans-mix-most-chains-applebees-olive-garden-2023-8) with people of different socioeconomic classes? 
- [ ] Any keyboard shortcuts for a GUI-heavy user?
- [ ] What does your productivity workflow look like? Which apps do you use? How does it help with organizing, planning, and focusing on getting things done? Do you have any automation tips and tricks? 
- [ ] If you were a recent college graduate in 2024, how would you utilize AI tools to aid your job search? How would you go about building your personal brand online? How would you structure your initial LinkedIn messages to make cold connections? Lastly, who would you choose to direct message first in your LinkedIn network?
- [ ] What are your thoughts on the current job market and layoffs in 2024, particularly in the tech industry where FAANG companies previously saw a boom in growth?
- [ ]  Given the current situation with the COVID-19 pandemic causing a significant increase in hiring followed by a market correction resulting in massive tech layoffs, how has your outlook changed regarding your dream job as a programmer in the tech industry? What advice would you give high school students who are considering pursuing a computer science degree? Additionally, what practical skills(trade skills), in addition to a technical background, should one consider learning?
- [ ] What steps can you take to ensure job security? How can you reduce job insecurity? Do you have any advice for managing your emotional well-being during the job search?
- [x] What do you like to do when you are procrastinating? 
- [ ] What is is museum concept you would love to visit? 
- [ ] What is a dish you didn't think you'd enjoy, but after tasting it, you fell in love with?

**Comment by meg-gutshall** [comment]
## Backpocket

### Response Time Etiquette

- [ ] How long is too long to wait to reply to a message? Is there such a thing as too soon? Do the rules change depending on which medium you're using to communicate? Do the rules change when it's work-related versus purely social?

### Emotional Rubber Ducking

- [x] Do you feel better after venting to someone about a problem? Do you like to deal with problems on your own? Do you prefer that the other person just listen, or that you have a conversation about whatever's bothering you?

**Comment by meg-gutshall** [comment]
# Recycle Bin Backpocket Topics

These topics were proposed in previous Coffees. They were either not discussed due to preference given to member-led discussion or were extremely popular topics and should be revisited!

## Undiscussed Backpocket Topics

- [ ] Have you ever had a skip level (1:1 with the manager above yours)? What did it look like? What kind of questions did you ask? How did you make sure it was productive?
- [x] What's a topic or idea you've been wanting to write about or research more deeply? What's been holding you back to do it? What's your comfort level with creating content?
- [ ] What are your tech devices? Do you use any watches or wearables? Do you ever work off your phone or tablet? Are there any tasks that you can do on multiple devices but you prefer one over the other (because millennials won't fill out a form on a phone)?
- [x] How do you organize your projects? Elaborate setup? A pile of folders in documents? Do you organize by category, time, etc.? How do you keep track of the work you've done?
- [ ] **Git:** How comfortable are you with git? What tools do you use to work with it? If you feel competent with it, how did you improve? Do you use the command line or a GUI?
- [ ] When are you most creative? What inspires your creativity?
- [ ] **Project planning:** What's your experience with it? Have you ever had to plan out a large project from start to finish? What tools did you use to accomplish this task? What components were critical to getting it done?
- [ ] **Context switching:** How do you handle productivity and context switching? Do you take notes, time-block to decrease the amount of switching, or something else?

## Popular Previously Discussed Backpocket Topics

- [ ] **Date: 2024.11.12** – What do you feel like is missing in your job or the tech industry right now?
- [ ] **Date: 2024.11.26** – How do the advancements in AI impact community dynamics? Do you see a change in quality v. quantity? How can it help to democratize contributions?
- [ ] **Date: 2024.12.03** – Do you think it's important for tech companies to have a developer community? What are the pros and cons? Who has good communities?
- [ ] **Date: 2024.12.10** – What do you care about most when it comes to tech? Are you a clock in/clock out person? Do you enjoy being up to date on all the latest happenings? What keeps you motivated either way?
- [ ] **Date: 2024.12.19** – **Positive talk:** Do you have any quotes, mantras, adages, axioms, idioms, thoughts, phrases, affirmations, or generally encouraging words that you use to motivate yourself or stay on track?
- [ ] **Date: 2025.01.09** – Recent data said that StackOverflow usage is down 77% due to the rise of AI solutions. Are you still using StackOverflow and other traditional searches? Are you fully on LLM solutions? If you use a mix, what do you prefer different searches for?
- [ ] **Date: 2025.01.23** – **Monotony:** What are your best tips and techniques for getting through super boring tasks? How do you break work up to make it more exciting? What type of work bores you the most?
- [ ] **Date: 2025.01.28** – **Problem Solving:** How do you come up with new ideas or approaches to solving problems? What's your process for working through the problem? Do you use any tools to help you brainstorm?
- [ ] **Date: 2025.02.11** – **The Rise of Tech-Based Creatives:** In recent years, we’ve seen a growing number of creative professionals (artists, musicians, writers, and filmmakers) using digital tools to produce, share, and evolve their work in innovative ways. From digital art platforms to AI-assisted music creation and virtual filmmaking, technology is enabling new forms of artistic expression that were once unimaginable. How is technology empowering creatives to push boundaries and reach new audiences? What role does the integration of engineering and creativity play in this new wave of art? Are we seeing a future where the lines between creativity and tech engineering are increasingly blurred?
